### PR TITLE
[Chore] Update navigation for speed stuff

### PR DESCRIPTION
**Pull request by kodster28**

### CODE REVIEW

File: content/railgun/user-guide/set-up/configuration-activation.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ $ sudo -u railgun /usr/bin/rg-listener -config=/etc/railgun/railgun.conf
 Once you have configured Railgun, you can test its operation on the Cloudflare dashboard:
 
 1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/) and select your account and domain.
-2. Go to **Speed** > **Optimization**.
+2. GGo to **Speed** > **Optimization** > **Other**..
```
Comment (kodster28) [inline content/railgun/user-guide/set-up/configuration-activation.md]:
```suggestion
2. Go to **Speed** > **Optimization** > **Other**.
```

Comment (jason-cf) [inline content/railgun/user-guide/set-up/configuration-activation.md]:
```suggestion
2. Go to **Speed** > **Optimization** > **Other**..
```

File: content/pages/platform/debugging-pages.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ If you are not using Functions or have reviewed that your Functions configuratio
 
 If your custom domain is proxied (orange-clouded) through Cloudflare, your zone's settings such as Auto Minify and caching will apply.
 
-If you are experiencing issues with a framework, like Nuxt.js, only on the custom domain, review if Auto Minify is enabled (log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login) > **Speed** > **Optimization** > **Auto Minify**) for HTML and disable it.
+If you are experiencing issues with a framework, like Nuxt.js, only on the custom domain, review if Auto Minify is enabled (log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login) > **Speed** > **Optimization** > **Content Optimization**. > **Auto Minify**) for HTML and disable it.
```
Comment (kodster28) [inline content/pages/platform/debugging-pages.md]:
```suggestion
If you are experiencing issues with a framework, like Nuxt.js, only on the custom domain, review if Auto Minify is enabled (log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login) > **Speed** > **Optimization** > **Content Optimization** > **Auto Minify**) for HTML and disable it.
```

File: content/railgun/user-guide/set-up/configuration-activation.md (hunk 1 above)
Comment (jason-cf) [inline content/railgun/user-guide/set-up/configuration-activation.md]:
```suggestion
2. Go to **Speed** > **Optimization** > **Other**..
```

File: content/support/Speed/Optimization - File Size/Using Cloudflare Auto Minify.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ To enable Auto Minify on your website:
 
 1.  Log into the [Cloudflare dashboard](https://dash.cloudflare.com/login).
 2.  Select your account and website.
-3.  Go to **Speed** > **Optimization**.
+3.  Go to **Speed** > **Optimization** > **Content Optimization**.=.
```
Comment (kodster28) [inline content/support/Speed/Optimization - File Size/Using Cloudflare Auto Minify.md]:
```suggestion
3.  Go to **Speed** > **Optimization** > **Content Optimization**.
```

File: content/support/Speed/Optimization - Mobile/Understanding Cloudflare Mobile Redirect.md [hunk 4]
```diff
@@ -23,29 +19,17 @@ ___
...
-{{<Aside type="note">}}
-**Keep Path** will redirect all mobile requests to the same resource
-hosted on your mobile-optimized website. **Drop Path** will redirect all
-mobile requests to your mobile-optimized website homepage.
-{{</Aside>}}
+1. Log into the [Cloudflare dashboard](https://dash.cloudflare.com).
+2. Select your account and zone.
+3. Go to **Speed** > **Optimization** > **Other**.
+4. For **Mobile Redirect**, select the subdomain that you would like to redirect from the drop down list.
+5. Choose **Keep Path** or **Drop Path**_._
```
Comment (kodster28) [inline content/support/Speed/Optimization - Mobile/Understanding Cloudflare Mobile Redirect.md]:
```suggestion
5. Choose **Keep Path** or **Drop Path**.
```

File: content/automatic-platform-optimization/get-started/activate-cf-wp-plugin.md [hunk 5]
```diff
@@ -10,10 +10,11 @@ For users on the free plan, be sure to purchase APO before installing the WordPr
 
 ## Purchase APO
 
-1.  Open your Cloudflare dashboard.
-2.  Select the **Speed** tile and locate the **Optimized Delivery** section.
-3.  From **Automatic Platform Optimization for WordPress**, select **Purchase**.
-4.  Enter your payment information and select **Confirm payment**.
+1.  Log into the [Cloudflare dashboard](https://dash.cloudflare.com).
+2. Select your account and zone.
```
Comment (jason-cf) [inline content/automatic-platform-optimization/get-started/activate-cf-wp-plugin.md]:
```suggestion
2.  Select your account and zone.
```

Comment (jason-cf) [reply content/automatic-platform-optimization/get-started/activate-cf-wp-plugin.md]: @kodster28 check spacing

File: content/fundamentals/speed/prefetch-urls.md [hunk 6]
```diff
@@ -28,7 +28,7 @@ For Cloudflare to start prefetching URLs, you will need to [enable the feature](
 To enable **Prefetch URLs** in the dashboard:
 
 1.  Log in to your [Cloudflare account](https://dash.cloudflare.com) and go to a specific domain.
-2.  Go to **Speed** > **Optimization**.
+2. Go to **Speed** > **Optimization** > **Content Optimization**.
```
Comment (jason-cf) [inline content/fundamentals/speed/prefetch-urls.md]: spacing again

Comment (kodster28) [reply content/fundamentals/speed/prefetch-urls.md]: From what I can see, the spacing at the start of an `ol` doesn't affect the rendering at all.

File: content/support/Speed/Optimization - Delivery/Configuring Cloudflare Mirage.md [hunk 7]
```diff
@@ -50,11 +50,10 @@ ___
 
 To enable Mirage image optimization for your domain,
 
-1.  Log in to your Cloudflare account.
-2.  Navigate to the **Speed** tab. 
-3.  Click the **Optimization** tab.
-4.  Scroll down to the **Optimized Delivery** section.
-5.  Toggle the Mirage switch to _On_.
+1.  Log in to the [Cloudflare dashboard](https://dash.cloudflare.com).
+2. Select your account and zone.
```
Comment (jason-cf) [inline content/support/Speed/Optimization - Delivery/Configuring Cloudflare Mirage.md]: spacing

**Review by jason-cf — APPROVED** [review]

**Pull request merged**